NapDatabaseExport/SVExportProviderBase.cs

```diff
@@ -12,7 +13,7 @@ public abstract class SVExportProviderBase : ExportProvider
 
         public override void StartExport (string filePath)
         {
-            writer = File.CreateText (filePath);
+            writer = new StreamWriter(filePath, false, new UTF8Encoding(true));
```

Comment:
Fix the spacing

Reply:
Is this written in Mono Develop or some other 3-rd party "fancy" editor?

Reply:
Visual studio 2017 and 2019